hack/release/create.sh

```diff
@@ -23,7 +23,7 @@ set -o pipefail
 
 if [ "$#" -ne 2 ]; then
     echo "Usage: create.sh release-version next-prerelease-version"
-    exit -1
+    exit 1
```

Comment:
@BenTheElder  I think that you are the expert here :)

Reply:
_technically_ 255 would be the equivalent code on many platforms (the spec only mentions 8 bits and signed -> unsigned -1 -> 255), but in this case we're just setting any non-zero exit code so 1 SGTM :-)